lms/static/sass/shared/_fields.scss

```diff
@@ -0,0 +1,69 @@
+// lms - shared - fields
+// ====================
+
+
+.u-field {
+    padding: $baseline 0;
+    border-bottom: 1px solid $gray-l5;
+
+    .message-error {
+        color: $alert-color;
+    }
+
+    .message-validation-error {
+        color: $warning-color;
+    }
+
+    .message-in-progress {
+        color: $gray-d2;
+    }
+
+    .message-success {
+        color: $success-color;
+    }
+}
+
+.u-field-readonly {
+    input[type="text"],
+    input[type="text"]:focus {
+        background-color: transparent;
+        padding: 0px;
+        border: none;
+        box-shadow: none;
+    }
+}
+
+.u-field-title {
+    width: flex-grid(3, 12);
+    display: inline-block;
+    color: $dark-gray1;
+    vertical-align: top;
+    margin-bottom: 0;
```

Comment:
I'm seeing an odd behavior where the field title / label is clickable. This is helpful in the case of input fields (as the focus is then set to the input value) but isn't useful for the other cases. My suggestion here would be to add a pointer-events: none; property to this class, but I wanted to check with @explorerleslie, @cptvitamin in case this was previously requested as a change for a11y reasons. 

Once we get confirmation from them about this being ok either way this can be updated. Thanks!